src/Datadog.Trace.ClrProfiler.Managed/Emit/MethodBuilder.cs

```diff
@@ -64,6 +66,12 @@ public MethodBuilder<TDelegate> WithConcreteTypeName(string typeName)
             return this.WithConcreteType(concreteType);
         }
 
+        public MethodBuilder<TDelegate> WithSimpleNameFilters(params string[] namespaceNameFilters)
```

Comment:
I'm not sure what "simple name" means in this context. Consider calling this `WithNamespaceFilters()`.